float64 error_distancia
float64 error_angular